rocky/account/mixins.py

```diff
@@ -89,7 +89,16 @@ def setup(self, request, *args, **kwargs):
                 user=self.request.user, organization=self.organization
             )
         except OrganizationMember.DoesNotExist:
-            raise Http404()
+            if not self.request.user.is_superuser:
+                raise Http404()
+
+            self.organization_member = OrganizationMember(
+                user=self.request.user,
+                organization=self.organization,
+                status=OrganizationMember.STATUSES.ACTIVE,
+                trusted_clearance_level=4,
+                acknowledged_clearance_level=4,
```

Comment:
Maybe the clearance level shouldn't be acknowledged by default?

Reply:
Perhaps, although this is also the default now for organisation members that get created for superusers when they add an organisation:
<img width="990" alt="Schermafbeelding 2023-08-31 om 14 36 01" src="https://github.com/minvws/nl-kat-coordination/assets/46660228/f1d32dd0-5d3d-49dd-9028-17cb4f7cc7ea">

Reply:
One could argue that creating an organisation yourself gives you different permissions than joining an existing one, but maybe we should also remove the acknowledged clearance level from `views/organization_add.py`?

Reply:
Yes, the minimal extra steps superusers then need to take also make sure they're aware they're dealing with a new organization. If @underdarknl agrees I can set the default `acknowledged_clearance_level` to 0 in both scenario's.

Reply:
I think this depends on ones understanding/interpretation of what a superuser is or should be. Afaik someone who is superuser should be able to access all organisations and should be able to perform all actions. I have no real reference for this, but I think that it is why a superuser gets an acknowledged level of 4 by default.